README.md

```diff
@@ -181,7 +181,6 @@
 
 ## Videos
 
-- [The New Front-end Stack. JavaScript, APIs and Markup](https://vimeo.com/163522126) - Matt Biilmann.
```

Comment:
I believe we should not remove valid links to avoid awesome-lint errors. It's related with timeout while trying to ping them. See https://github.com/sindresorhus/awesome-lint/issues/50

Reply:
I'll revert that commit.